### PR TITLE
cryptol-remote-api server reset feature

### PR DESCRIPTION
**Pull request by pnwamk**

### CODE REVIEW

File: cryptol-remote-api/python/README.md [hunk 1]
```diff
@@ -1,83 +1,190 @@
-# Cryptol Python Package
+# Cryptol Python Client
 
-In-development Python client for Cryptol's RPC server.
+In-development Python client for Cryptol. Currently tested on Linux and MacOS --
+at present there may be some minor issues running the Python Client in a Windows
+environment that needs addressed (e.g., some Python process management methods
```
Comment (atomb) [inline cryptol-remote-api/python/README.md:5]: "needs addressed" -> "need to be addressed"

File: cryptol-remote-api/python/README.md [hunk 2]
```diff
@@ -1,83 +1,190 @@
...
+## Acquiring a Cryptol Server
+
+There are several ways a server executable can be obtained.
+
+### Server executables
+
+An executable of the server is now included in each release of Cryptol.
+
+Nightly server builds can be found under the 
+[Packages](https://github.com/orgs/GaloisInc/packages?repo_name=cryptol) section
```
Comment (atomb) [inline cryptol-remote-api/python/README.md:107]: The packages here include only Docker images. The nightly builds are here: https://github.com/GaloisInc/cryptol/actions/workflows/nightly.yml

**Review by atomb — APPROVED** [review]
Looks great! There are a couple of small tweaks to the README that I noted, but I think it's otherwise good to go.

File: cryptol-remote-api/python/README.md (hunk 2 above)
Comment (pnwamk) [reply cryptol-remote-api/python/README.md:113]: Ah good point -- I've added pointers to both now.

**Pull request merged**